src/main/java/edu/osu/waiting4ubackend/client/UserDBClient.java

```diff
@@ -69,7 +73,43 @@ public User getUserById(long id) {
                 .setUserName(userEntity.getString("userName"))
                 .setEmail(userEntity.getString("email"))
                 .setIntroduction((userEntity.getString("introduction")))
+                .setPassword(userEntity.getString("password"))
+                .setPreferences(DBClientHelper.convertToList(userEntity.getList("preferences")))
+                .build();
+    }
+
+    public void updatePreferences(String newPreference, long id) {
```

Comment:
I didn't see this 'updatePreferences' method is called.

Reply:
updated